doc/classes/OS.xml

```diff
@@ -23,7 +23,7 @@
 			<return type="void" />
 			<description>
 				Shuts down the system MIDI driver. Godot will no longer receive [InputEventMIDI]. See also [method open_midi_inputs] and [method get_connected_midi_inputs].
-				[b]Note:[/b] This method is implemented on Linux, macOS, and Windows.
+				[b]Note:[/b] This method is implemented on Linux, macOS, Web, and Windows.
```

Comment:
```suggestion
				[b]Note:[/b] This method is implemented on Linux, macOS, Windows, and Web.
```

Reply:
This kind of statement tends to sort the platforms alphabetically, so the existing note is fine too.

Reply:
Yes, but I don't think we need to inject Web inside desktop OSes.

Reply:
We often just add things at the end rather than sorting them as well